specification/purview/data-plane/Azure.Analytics.Purview.Scanning/stable/2023-09-01/scanningService.json

```diff
@@ -0,0 +1,10292 @@
+          },
+          {
+            "in": "path",
+            "name": "scanName",
+            "required": true,
+            "type": "string",
+            "description": "scan name"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success.",
+            "schema": {
+              "$ref": "#/definitions/Trigger"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Scanning Service.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Triggers_Disable": {
+            "$ref": "./examples/Triggers_Disable.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AzureKeyVaultProperties": {
+      "type": "object",
+      "properties": {
+        "baseUrl": {
+          "type": "string"
+        },
+        "description": {
+          "type": "string"
+        }
+      }
+    },
+    "ProxyResource": {
+      "description": "The proxy resource.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The resource identifier.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "The resource name.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "AzureKeyVault": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "allOf": [
+            {
+              "$ref": "#/definitions/AzureKeyVaultProperties"
+            }
+          ]
+        }
+      }
+    },
+    "ErrorModel": {
+      "description": "The error model.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "A unique error code that identifies the specific error.",
+          "type": "string"
+        },
+        "message": {
+          "description": "A human-readable error message that provides more details about the error.",
+          "type": "string"
+        },
+        "target": {
+          "description": "The specific component that the error is associated with.",
+          "type": "string"
+        },
+        "details": {
+          "description": "An array of nested ErrorModel objects that provides additional error details.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ErrorModel"
+          }
+        }
+      }
+    },
+    "ErrorResponseModel": {
+      "description": "The error response model.",
+      "type": "object",
+      "properties": {
+        "error": {
+          "allOf": [
+            {
+              "$ref": "#/definitions/ErrorModel"
+            }
+          ]
```

Comment:
could you write this as a normal json reference (instead of allOf)?

Reply:
Sure, updated. The "allOf" was generated by some extension, so there are a lot of them